### DISCUSSION TITLE
evince: Postscript support

### DISCUSSION THREAD

**Discussion by kriztioan** [Homebrew/discussions#6] [Everyday usage] [answered]
A while back evince disabled postscript support by default (https://github.com/GNOME/evince/blob/10842b31d5fc714115eb90892f68190609033954/NEWS#L781). The necessary spectre-library is already required by the formula. Simple adding the '-Dps' flag under install/flags in the formula and rebuilding from source re-enables Postscript support. Please update the tap and formula to re-enable Postscript support by default. Thanks!

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request should help you get started with that.

**Reply by kriztioan** [reply]
Thank you for the link, I'll give it a go!
